### ISSUE TITLE
Command to run already built image

### ISSUE THREAD

**Issue by saulshanabrook** [jupyter/repo2docker#632]
Often, I would like to play with what volumes are mounted, or what command is executed, without having to rebuild the docker image.

This takes a long time when working on a JupyterLab extension that depends on JupyterLab master and so has to rebuild all of JupyterLab every image build.

Currently, I am looking for the image name manually with `docker image ls` and running it manually. This isn't ideal, because then it isn't run in the same way as `repo2docker` is running it. You pass some extra options on start, like a port and the custom display URL. I would like to keep all these nicetities and not have to hunt in the image list for the most recent one.

I thought this is what the `--no-build` argument would do, but I guess it not only doesn't build but doesn't start. For example, [`docker-compose up`](https://docs.docker.com/compose/reference/up/) has a `--no-build` argument that won't build the images, but will start up the services.

**Comment by betatim** [comment]
This would be a great feature to have. I think it would make a good first issue for people who are already familiar with docker, the concept of images and containers.

A way to get started is to look at the mechanism used to generate the docker image names used by repo2docker, check if there is already one (with a similar name) and then launch that instead when the user specifies a particular command-line flag.

If someone wants to work on this please open a PR. It doesn't have to be perfect and it is even encouraged that you open a PR long before you think it is ready as a way to get feedback and help when needed.

**Comment by tmtaybah** [comment]
Hi, can I work on this issue?

**Comment by betatim** [comment]
Yes please! A good way to "claim" an issue is to start work on it and soon after open a Pull Request with `[WIP] ` at the start of the title. That way we know ho is working on what. The WIP tag means "this is still work in progress, not ready to be reviewed".

**Comment by betatim** [comment]
@saulshanabrook does the new docker build caching (fast builds if you don't edit `environment.yml`) and `--editable` solve your use case?
